StripeCore/StripeCore/Source/Analytics/STPAnalyticEvent.swift

```diff
@@ -78,12 +78,8 @@ import Foundation
     // MARK: - PaymentSheet Show
     case mcShowCustomNewPM = "mc_custom_sheet_newpm_show"
     case mcShowCustomSavedPM = "mc_custom_sheet_savedpm_show"
-    case mcShowCustomApplePay = "mc_custom_sheet_applepay_show"
-    case mcShowCustomLink = "mc_custom_sheet_link_show"
     case mcShowCompleteNewPM = "mc_complete_sheet_newpm_show"
     case mcShowCompleteSavedPM = "mc_complete_sheet_savedpm_show"
-    case mcShowCompleteApplePay = "mc_complete_sheet_applepay_show"
-    case mcShowCompleteLink = "mc_complete_sheet_link_show"
```

Comment:
How are we thinking about how removing these events could impact dashboards? Do we not use them?

Reply:
Oops, I commented on this exact question but didn't hit submit. We never sent these and never can. 'newpm' and 'savedpm' correspond to the two screens that PaymentSheet/FC can show.  There's no "Apple Pay" or "Link" screen.